modules/ROOT/pages/insight.adoc

```diff
@@ -61,6 +62,18 @@ See xref:mule-runtime::batch-processing-concept.adoc[Batch Processing].
 Ensure that this limitation doesn't violate your compliance requirements.
 * Logs for applications deployed to on-premises servers contain event information but not message contents.
 
+* Insight returns consistent results only when searching for terms that are an exact match.
++
+--
+Because Insight limits the number of terms that are included in a query, it might not find all matches when searching for a pattern.
+
+To work around this limitation, either:
```

Comment:
```suggestion
To work around this limitation, use one of these two methods:
```
This may be unimportant, but it sounds odd to have "To do a b and c, either: - Do x. - Do y." with no "or" anywhere. This rephrasing solves that. Accept or discard as you please.